src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export function Person({ person }) {
+  const {
+    name = 'no name',
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age ? <p className="Person__age">{`I am ${age}`}</p> : ''}
+
+      <p className="Person__partner">
+        {(isMarried
+          && `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`)
+          || `I am not married`
+        }
```

Comment:
```suggestion
        {(isMarried
          ? `${partnerName} is my ${partner}`)
          : `I am not married`
        }
```
It is better to use ternary operator here. Also, would be great to create a variable for partner before to make your code cleaner